### PR TITLE
Write First Unit Tests for Futarchy.sol

### PR DESCRIPTION
**Pull request by emaG3m**

### CODE REVIEW

File: test/app.js [hunk 1]
```diff
@@ -18,37 +24,212 @@ const FutarchyOracleFactory = artifacts.require('FutarchyOracleFactory.sol')
...
+      await futarchy.newDecision(script, 'abc')
+      expect((await futarchy.newDecision.call(script, metadata)).toNumber()).to.equal(1)
+    })
+  })
+
+  describe('getDecision()', async () => {
+    let script, metadata, returnValue, currentBlockNumber
+    beforeEach(async () => {
+      script = 'QmWmyoMoctfbAaiEs2G46gpeUmhqFRDW6KWo64y5r581Vz'
+      metadata = 'Give voting rights to all kitties in the world'
```
Comment (jpkcambridge) [inline test/app.js]: What about the moon kitties?

**Comment by jpkcambridge** [comment]
@emaG3m looks good to me! 👍

**Comment by mikec** [comment]
@emaG3m looking great! Really nice job getting the kernel/acl stuff from aragon to work! 👍 🎉

File: test/app.js (hunk 1 above)
Comment (emaG3m) [reply test/app.js]: :P

File: test/app.js [hunk 2]
```diff
@@ -18,37 +24,212 @@ const FutarchyOracleFactory = artifacts.require('FutarchyOracleFactory.sol')
...
+
+  const root = accounts[0]
 
   before(async () => {
-    const token = await MiniMeToken.new(NULL_ADDRESS, NULL_ADDRESS, 0, 'n', 0, 'n', true)
-    const centralizedOracleMaster = await CentralizedOracle.new()
-    const centralizedOracleFactory = await CentralizedOracleFactory.new(centralizedOracleMaster.address)
-    let { logs } = await centralizedOracleFactory.createCentralizedOracle("QmWmyoMoctfbAaiEs2G46gpeUmhqFRDW6KWo64y5r581Vz")
-    const futarchyOracleFactory = await deployFutarchyMasterCopies()
+    const kernelBase = await getContract('Kernel').new(true) // petrify immediately
```
[the remark this answers is not in the document]
Comment (emaG3m) [reply test/app.js]: no idea, but I'm going to keep it in case it's useful later lol

**Pull request merged**